test/services/authenticator.test.js

```diff
@@ -1,84 +1,68 @@
-const fs = require('fs');
-const jwt = require('jsonwebtoken');
-const fsExtra = require('fs-extra');
 const authenticator = require('../../src/services/authenticator');
-
-const key = 'test-token-key';
-const tokenPath = './test/services/tokens';
-const storeToken = (fileName, expiresIn = '14 days') => {
-  const token = jwt.sign({}, key, { expiresIn });
-  fs.writeFileSync(`${tokenPath}/${fileName}`, token);
-  return token;
-};
```

Comment:
we just steal the code from here to set it into test-cli.
Like that, methods can de reused in test-cli-mock-token.